src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -38,13 +38,27 @@ export class ClaimAmountRow {
     return new ClaimAmountRow(undefined, undefined)
   }
 
+  static translateToNumber (input?: any): any {
```

Comment:
Can we use more precise types? I would expect `(input?: string): number`.

Reply:
return type can be a number or string (original value)